codespell_lib/data/dictionary.txt

```diff
@@ -16599,12 +16639,14 @@ ges->goes, guess,
 gess->guess
 get's->gets
 get;s->gets
+geted->gated, got, gotten,
 getfastproperyvalue->getfastpropertyvalue
 getimezone->gettimezone
 geting->getting
 getlael->getlabel
 getoe->ghetto
 getoject->getobject
+getted->got, gotten, getter, vetted, netted,
```

Comment:
Perhaps "betted"? https://www.merriam-webster.com/dictionary/betted

Reply:
Added.

Reply:
Actually, not possible:
```
 error getted: correction betted is an error itself in the same dictionary file dictionary.txt
```

Reply:
> Actually, not possible:
> 
> ```
>  error getted: correction betted is an error itself in the same dictionary file dictionary.txt
> ```

Should we keep that correction? Hmm.